rosidl_generator_py/resource/msg.py.template

```diff
@@ -0,0 +1,27 @@
+# Generated by rosidl_generator_py
```

Comment:
Please use similar comments at the top of the templates as in the other generators (exact template, copyright notice).